source/_components/google_assistant.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: "Google Assistant"
+description: "Setup for Google Assistant integration"
+date: 2017-10-17 12:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google-assistant.png
+ha_category: Voice
+featured: true
+ha_release: 0.56
+---
+
+# Google Assistant Docs
+The Google Assistant component requires a bit more setup than most due to the way Google requires Assistant Apps to be set up.
+
+### {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+google_assistant:
+  project_id: someproject-2d0b8
+  client_id: [long random string]
+  access_token: [different long random string]
+  exposed_domains:
+    - switch
+    - light
+    - group
+```
+
+*Note:* It's very important that you use something very long strings for `client_id` and `access_token`. Those are essentially credentials to your Home Assistant instance.
+
+*Configuration Variables:*
+* *expose_by_default* (Optional): Expose devices in all supported domains by default.
+* *project_id* (Required): Project ID from the Google Developer console (looks like `words-2ab12`)
+* *client_id* (Required): A long random url safe string (no spaces or special characters) that will be used for Implicit OAuth.
+* *access_token* (Required): Another long random url safe string.
+* *exposed domains* (Optional): An array of Home Assistant domains to expose to Google Assistant.
+
+You can also customize your devices similar to other components by adding keys to entities:
+
+```yaml
+homeassistant:
+  customize:
+    master_bedroom_light:
+      google_assistant: true
+      google_assistant_name: bedroom light
+    bedroom_blinds:
+      aliases:
+        - bedroom shades
+        - bedroom covers
+```
+
+*Entity Customization Keys:*
+* *google_assistant*: True exposes entity, false will hide it
+* *google_assistant_name*: Can be used to override the primary name of an entity. By default the `friendly_name` of an entity is used.
+* *aliases*: Provides "nicknames" to Google Assistant. These function as alternate names for an entity that Assistant will understand when spoken.
+
+### {% linkable_title Setup %}
+
+1. Install the [gactions CLI](https://developers.google.com/actions/tools/gactions-cli) (you'll use this later)
+2. Create a new file named `project.json` and replace the hostname with your hostname in the URL. Note: This must be an HTTPS URL to work.
```

Comment:
What is a hostname? Assume that people know nothing :-)

"and replace "example.com" with the url of your HomeAssistant instance"

or something along those lines?